openshift/release/update-to-head.sh

```diff
@@ -24,12 +24,8 @@
 set -e
 REPO_NAME=$(basename $(git rev-parse --show-toplevel))
```

Comment:
Not your change but here unwanted splicing may happen `$(git rev-parse --show-toplevel)` should be in `""`.

Reply:
updated